cmake/GzPython.cmake

```diff
@@ -43,3 +43,6 @@ endif()
 if(Python3_EXECUTABLE AND NOT PYTHON_EXECUTABLE)
   set(PYTHON_EXECUTABLE ${Python3_EXECUTABLE})
 endif()
+
+set(IGN_PYTHON_VERSION ${GZ_PYTHON_VERSION} CACHE STRING  # TODO(CH3): Deprecated. Remove on tock.
+  "Deprecated. Use [GZ_PYTHON_VERSION] instead! Specify specific Python version to use ('major.minor' or 'major')")
```

Comment:
Should this be up top?

Reply:
https://github.com/gazebosim/gz-cmake/pull/275/commits/5f4f308a2aa9b85caeaeab20d15cad690e29e545